### PR TITLE
Format the codebase and fix shellcheck

### PR DESCRIPTION
**Pull request by fpetkovski**
This PR formats the codebase and fixes shellcheck in the CI.

* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Review by jan--f — CHANGES_REQUESTED** [review]
This seems to break `hack/generate-versions.sh`

Also if we want the scripts in hack shellcheck'ed, should we add that to `hack/shellcheck.sh`?

**Comment by fpetkovski** [comment]
Good catch. `generate-versions` does not pass shellcheck and I tried to fix it, but it turns out I committed the wrong "fix". 

The scripts in `hack` already seem to be shellcheck-ed, this is how I found the issue in `generate-versions`. Did you have something else in mind?

File: pkg/manifests/manifests.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/openshift/cluster-monitoring-operator/pkg/promqlgen"
 	"github.com/pkg/errors"
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"golang.org/x/crypto/bcrypt"
```
Comment (slashpai) [inline pkg/manifests/manifests.go:34]: This change is covered in https://github.com/openshift/cluster-monitoring-operator/pull/1363/files#diff-1f9d2d18d4cabd8262e585b932330183f7cb9b544ac9f27c8bcb3ef9c8d14376R34   :)

**Comment by jan--f** [comment]
> The scripts in hack already seem to be shellcheck-ed, this is how I found the issue in generate-versions. Did you have something else in mind?

hmm no, just wondering how that got through our CI then, as that runs a required shellcheck job :thinking:

**Comment by fpetkovski** [comment]
We never verified it in the CI :) But we should now: https://github.com/openshift/release/pull/21638

File: pkg/manifests/manifests.go (hunk 1 above)
Comment (fpetkovski) [reply pkg/manifests/manifests.go:34]: Good catch. 
I don't think there is a problem with having this change in two PRs, it should not cause any conflicts. 
Also, I would like to enable go linting checks in the CI and for that PR to be merged, the codebase has to be properly formatted.

**Review by jan--f — CHANGES_REQUESTED** [review]
Imho it would be better to run `install` in a sub shell, instead of explicitly cd'ing back. `cd -` is quite dependent on context whereas  `(install)` will leave the context of the main script untouched.

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by fpetkovski** [comment]
@jan--f good idea, done

**Comment by fpetkovski** [comment]
/label px-approved
no px needed

/label docs-approved
no user-facing docs needed

/label qe-approved
no qe needed

**Pull request merged**